timber/src/test/java/timber/log/TimberJavaTest.java

```diff
@@ -1,30 +0,0 @@
-package timber.log;
-
-import org.junit.Test;
-
-import static org.junit.Assert.fail;
-
-public class TimberJavaTest {
-  @SuppressWarnings("ConstantConditions")
-  @Test public void nullTree() {
-    try {
-      Timber.plant((Timber.Tree) null);
-      fail();
-    } catch (IllegalArgumentException ignored) {
```

Comment:
NPE

Reply:
So I should put these tests back and catch an NPE instead?

Reply:
Yea, i think the purpose of the test is to catch Java interop, so updating the test should be enough.

Reply:
Done